migrations/2_deploy_ingress_contract.js

```diff
@@ -12,11 +12,12 @@ module.exports = (deployer, network) => {
 
     } else {
         // If supplied an address, make sure there's something there
-        const ingressInstance = Ingress.at(ingressAddress);
+        let ingressInstance = await Ingress.at(ingressAddress);
```

Comment:
These can be const, do we want them to be const?

Reply:
I missed a commit, in which I make it not const - to accommodate migrating to ganache where it is not fixed

Reply:
const it is